content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,53 @@
+| **[SLO Widgets][4] (up to 90 days of historical data)**                                                  | Available                                                                                                           | Available                                                                                                                           | Available                                                        |
+| [**SLO Data Source**][5]                                              | Available (with up to 15 months of historical data)                                                                | Not available                                                                                                                                     | Not available                                                |
+| **Handling missing data in the SLO calculation**                      | Missing data is ignored in SLO status and error budget calculations                                                                       | Missing data is handled based on the [underlying Monitor's configuration][6]                                                                                        | Missing data is treated as uptime in SLO status and error budget calculations        |
+| **Uptime Calculations**                                          |  N/A                                                                                  |  Uptime calculations are based on the underlying Monitor <br><br>If groups are present, overall uptime requires *all* groups to have uptime| [Uptime][7] is calculated by looking at discrete time chunks, not rolling time windows<br><br>If groups are present, overall uptime requires *all* groups to have uptime |
+| **Calendar View on SLO Manage Page**                                   | Available                                                                                                                                | Not available                                                                                                                                                      | Available                                                                            |
+| **Public [APIs][8] and Terraform Support**                                   | Available                                                                                                                                 | Available                                                                                                                                                     | Not available                                                                            |
```

Comment:
For time-slices can we say "Not yet available" or "Coming soon"?

Reply:
Same as above